package.json

```diff
@@ -114,7 +114,6 @@
     "ember-cli-deploy": "^1.0.2",
     "ember-cli-deploy-build": "^2.0.0",
     "ember-cli-deploy-git": "^1.3.4",
-    "ember-cli-deploy-git-ci": "^1.0.1",
```

Comment:
With GitHub Actions, we don't need `ember-cli-deploy-git-ci`. For more information, please see sections 3b and 3c of https://crunchingnumbers.live/2020/08/31/ci-with-github-actions-for-ember-apps-part-2/.